cmd/apps/openfaas_app.go

```diff
@@ -234,6 +254,7 @@ func MakeInstallOpenFaaS() *cobra.Command {
 			WithValuesFile(fmt.Sprintf("values%s.yaml", valuesSuffix)).
 			WithHelmURL("https://openfaas.github.io/faas-netes/").
 			WithHelmRepo("openfaas/openfaas").
+			WithHelmUpdateRepo(updateRepo).
```

Comment:
Good spot!

@Shikachuu / @Jasstkn could you check if we have missed this elsewhere in other apps?

Reply:
Not in the few latest changes but it's definitely missing from some of the repos

```
cat ./cmd/apps/* | grep "WithHelmURL" | wc -l
      39
      
cat ./cmd/apps/* | grep "WithHelmUpdateRepo(updateRepo)." | wc -l
      22
```